### PR TITLE
Example for SDL2 blog

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

**Comment by prince-chrismc** [comment]
I guess new ones do :)

File: examples/libraries/sdl2/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.15)
```
Comment (memsharded) [inline examples/libraries/sdl2/CMakeLists.txt:1]: put everything inside a "introduction" folder.
This way we can add more ``sdl2`` advanced examples in the future.

**Review by memsharded — COMMENTED** [review]
Looks good!

**Review by RubenRBS — APPROVED** [review]

**Review by czoido — APPROVED** [review]
Nice! Agree with the subfolder comment otherwise looking good!

**Comment by prince-chrismc** [comment]
I have no objection to the subfolder but we've never used it :)

Waiting on recipe https://github.com/conan-io/conan-center-index/pull/18366#issuecomment-1625465918 should be the last one

**Comment by memsharded** [comment]
> I have no objection to the subfolder but we've never used it :)

Yes, we have, check https://github.com/conan-io/examples2/tree/main/examples/libraries, all libraries examples live in a subfolder

**Comment by prince-chrismc** [comment]
Sorry I was not clear what I specifically replying to! Let me try again.

> advanced examples in the future.

We have never done this :) There is only ever been 1 introduction blog for each library

image

I think this making it more complicated for use to maintain especially given we want a larger breath of verticals
